requirements.txt

```diff
@@ -1,7 +1,7 @@
 APScheduler==3.6.3
 Flask==1.1.2
 Flask-Cors==3.0.9
-Flask-JWT-Extended==3.24.1
+Flask-JWT-Extended==3.25.0
```

Comment:
no need to make a separate change. This change was already fixed and merged through the latest commit to develop. If you pull --rebase the latest  change, this will be redundant.